### DISCUSSION TITLE
Set node --max_old_space_size to pnpm

### DISCUSSION THREAD

**Discussion by begandroide** [pnpm/pnpm#3717] [Q&A]
Hi! recently im migrating from npm to pnpm, and in my project i has node configured with --max_old_space_size=8048, but pnpm is not taking de setting. how i can set this option?

Regards!

**Comment by zkochan** [comment]
How do you configure node with --max_old_space_size=8048?

**Reply by begandroide** [reply]
Hi @zkochan!  I always use npm start from this script in package.json:
```json
{ 
    "scripts": {
        "start": "node --max_old_space_size=8048 ./node_modules/@angular/cli/bin/ng serve"
    }
}
```

When I try to use `pnpm`, I execute `pnpm start` (but has issue with max heap size). 

I tried to set --max_old_space_size in env vars but still with the issue of stack size. 

pd: `node` version: 12.17 , `pnpm` version: latest stable
